src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -477,7 +576,9 @@ void BaseQNAcceleration::iterationsConverged(
   // this has to be done in iterations converged, as PP won't be called any more if
   // convergence was achieved
   concatenateCouplingData(cplData, _dataIDs, _values, _oldValues);
-  updateDifferenceMatrices(cplData);
+  auto cplDataCopy = cplData;
```

Comment:
Why the copy?

Reply:
This transformed data is not transformed backward later like the other ones from the not converging iterations, since the backward transformation happens in(at the end of) the acceleration step and when this iteration converges, no acceleration is done. 
So I use the copied data to avoid influencing the later convergence measure calculation etc.